### PR TITLE
log4cpp 1.1.4

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

---

continuing #129891

### CODE REVIEW

File: Formula/log4cpp.rb [hunk 1]
```diff
@@ -25,12 +25,6 @@ class Log4cpp < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "fa93ce1f4cce44107a131667b2350814eb79a762c63c8bd4f35d283f21a25a10"
   end
 
-  # Fix -flat_namespace being used on Big Sur and later.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-pre-0.4.2.418-big_sur.diff"
-    sha256 "83af02f2aa2b746bb7225872cab29a253264be49db0ecebb12f841562d9a2923"
-  end
```
Comment (chenrui333) [inline Formula/log4cpp.rb:32]: this needs to be updated for 1.1.4 release

**Comment by chenrui333** [comment]
only failed on 13-arm, 🤔

**Comment by Bo98** [comment]
Maybe try `ENV.cxx11`?

**Comment by chenrui333** [comment]
> Maybe try `ENV.cxx11`?

👍 (however it works in my local without it)

**Review by carlocab — APPROVED** [review]

**Pull request merged**